scene/animation/animation_tree.h

```diff
@@ -194,6 +194,8 @@ class AnimationNodeEndState : public AnimationRootNode {
 class AnimationTree : public Node {
 	GDCLASS(AnimationTree, Node);
 
+	static void _call_object(Object *p_object, const StringName &p_method, const Vector<Variant> &p_params, bool p_deferred);
```

Comment:
It might not need to be `static`, I think it was just like this to be able to be kept as a helper function just in the .cpp.